contents/docs/feature-flags/common-questions.mdx

```diff
@@ -25,6 +25,27 @@ Local evaluation payloads are typically used for much, _much_ more than 10 users
 
 ---
 
+## My feature flags are too expensive, what can I do about this?
+
+Feature flags can get expensive if your code ends up sending multiple requests to get feature flags for each person.
+
+If this is happening on the client side, i.e. while using the posthog-js SDK, you have two options:
+
+1. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags_on_first_load` setting. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
+2. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags` setting. This stops automatically requesting feature flags. Instead, use the [bootstrap option](/docs/libraries/js#bootstrapping-flags) to load flags only once, via local evaluation, which makes no requests for flags from the client side.
```

Comment:
```suggestion
2. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags` setting. This stops automatically requesting feature flags. Instead, use the [bootstrap option](/docs/libraries/js#bootstrapping-flags) to load flags only once, via /docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation, which makes no requests for flags from the client side.
```

Reply:
added a newline and fixed link

Reply:
`As per` again, and kinda weird that it repeats itself?

Also I think it might be worthwhile to mention why someone wouldn't do this or why it's not the default, as this might just make people wonder why we don't just have everyone set up that way.